docker-compose.yml

```diff
@@ -46,7 +45,6 @@ services:
       - ./:/app:rw
       - frontend_core_build:/app/target
       - frontend_wasm_build:/app/wasm/target
-      - frontend_wasm_pkg:/app/wasm/pkg
```

Comment:
I removed this so that when the compiler runs inside the container, it updates the type bindings outside. Otherwise, you have to manually run the wasm compiler every time you make changes to update the type bindings.